bevy_lint/src/paths.rs

```diff
@@ -8,6 +8,8 @@
 
 /// <https://github.com/bevyengine/bevy/blob/release-0.16.0/crates/bevy_app/src/app.rs#L78>
 pub const APP: [&str; 3] = ["bevy_app", "app", "App"];
+/// <https://github.com/bevyengine/bevy/blob/v0.16.0/crates/bevy_render/src/camera/camera.rs#L346>
+pub const CAMERA: [&str; 4] = ["bevy_render", "camera", "camera", "Camera"];
 /// <https://github.com/bevyengine/bevy/blob/release-0.16.0/crates/bevy_ecs/src/system/commands/command.rs#L48>
```

Comment:
For a follow-up: I'm noticing half these links use the `release-0.16.0` branch and the other half use the `v0.16.0` tag. Even though they're the same commit, we should make these links consistent. (I'm slightly leaning towards `v0.16.0` for brevity's sake.)